ts/src/oxfun.ts

```diff
@@ -0,0 +1,2843 @@
+        //                 "side": "SELL",
+        //                 "amount": "200",
+        //                 "orderType": "MARKET",
+        //                 "createdAt": "1715592079986",
+        //                 "source": 11
+        //             }
+        //         ]
+        //     }
+        //
+        const data = this.safeList (response, 'data', []);
+        const order = this.safeDict (data, 0, {});
+        return this.parseOrder (order, market);
+    }
+
+    async createOrders (orders: OrderRequest[], params = {}): Promise<Order[]> {
+        /**
+         * @method
+         * @name oxfun#createOrders
+         * @description create a list of trade orders
+         * @see https://docs.ox.fun/?json#post-v3-orders-place
+         * @param {Array} orders list of orders to create, each object should contain the parameters required by createOrder, namely symbol, type, side, amount, price and params
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {int} [params.timestamp] *for all orders* in milliseconds. If orders reach the matching engine and the current timestamp exceeds timestamp + recvWindow, then all orders will be rejected.
+         * @param {int} [params.recvWindow] *for all orders* in milliseconds. If orders reach the matching engine and the current timestamp exceeds timestamp + recvWindow, then all orders will be rejected. If timestamp is provided without recvWindow, then a default recvWindow of 1000ms is used.
+         * @param {string} [params.responseType] *for all orders* FULL or ACK
+         * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        await this.loadMarkets ();
+        const ordersRequests = [];
+        for (let i = 0; i < orders.length; i++) {
+            const rawOrder = orders[i];
+            const symbol = this.safeString (rawOrder, 'symbol');
+            const market = this.market (symbol);
+            const type = this.safeString (rawOrder, 'type');
+            const side = this.safeString (rawOrder, 'side');
+            const amount = this.safeNumber (rawOrder, 'amount');
+            const price = this.safeNumber (rawOrder, 'price');
+            const orderParams = this.safeDict (rawOrder, 'params', {});
+            const orderRequest = this.createOrderRequest (market, type, side, amount, price, orderParams);
+            ordersRequests.push (orderRequest);
+        }
+        const request: Dict = {
+            'responseType': 'FULL',
+            'timestamp': this.milliseconds (),
+            'orders': ordersRequests,
+        };
+        const response = await this.privatePostV3OrdersPlace (this.extend (request, params));
+        const data = this.safeList (response, 'data', []);
+        return this.parseOrders (data);
+    }
+
+    createOrderRequest (market, type, side, amount, price = undefined, params = {}) {
```

Comment:
i would keep the signature the same as createOrder (receiving a symbol instead of market)

Reply:
Yeah, it is reasonable. I'll change it.